securitycenter/src/main/java/com/google/cloud/examples/securitycenter/snippets/NotificationConfigSnippets.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.examples.securitycenter.snippets;
+
+import com.google.cloud.securitycenter.v1p1beta1.CreateNotificationConfigRequest;
+import com.google.cloud.securitycenter.v1p1beta1.NotificationConfig;
+import com.google.cloud.securitycenter.v1p1beta1.NotificationConfig.EventType;
+import com.google.cloud.securitycenter.v1p1beta1.NotificationConfig.StreamingConfig;
+import com.google.cloud.securitycenter.v1p1beta1.NotificationConfigName;
+import com.google.cloud.securitycenter.v1p1beta1.OrganizationName;
+import com.google.cloud.securitycenter.v1p1beta1.SecurityCenterClient;
+import com.google.cloud.securitycenter.v1p1beta1.SecurityCenterClient.ListNotificationConfigsPagedResponse;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.FieldMask;
+import java.io.IOException;
```

Comment:
Imports should be in the snippet tag.

Reply:
I'm going to break these out into their our files to include imports.